src/main/java/com/notably/logic/parser/NotablyParser.java

```diff
@@ -98,6 +99,9 @@ public NotablyParser(Model notablyModel) {
 
         case ExitCommand.COMMAND_WORD:
             return List.of(new ExitCommand());
+        case SearchCommandParser.COMMAND_WORD:
+            new SearchCommandParser(notablyModel).parse(arguments);
+            return List.of();
```

Comment:
Hmmm I think it's better to `return new SearchCommandParser(notablyModel).parse(arguments)`, then return `List.of()` in your `SearchCommandParser`, wdyt? This way it's more consistent with the other commands.

Reply:
In fact, you can argue that the `SearchCommandParser` will never create any `List.of()`, since an error will always be thrown no matter what.